packages/story-editor/src/karma/fixture/fixture.js

```diff
@@ -713,102 +715,10 @@ class APIProviderFixture {
 
     // eslint-disable-next-line react/prop-types
     const Comp = ({ children }) => {
-      const getStoryById = useCallback(
-        // @todo: put this to __db__/
-        () =>
-          asyncResponse({
-            title: { raw: '' },
-            status: 'draft',
-            author: 1,
-            slug: '',
-            date: '2020-05-06T22:32:37',
-            date_gmt: '2020-05-06T22:32:37',
-            modified: '2020-05-06T22:32:37',
-            excerpt: { raw: '' },
-            link: 'http://stories.local/?post_type=web-story&p=1',
-            preview_link: 'http://stories.local/?post_type=web-story&p=1',
-            story_data: {
-              version: DATA_VERSION,
-              pages: this._pages,
```

Comment:
Note: this was change to an empty array. `this._pages` was never referenced in this component, but I'm happy to put this back in if we want to update `this._pages`